src/EditorFeatures/CSharpTest2/CSharpServicesTest.Debug.xunit

```diff
@@ -0,0 +1,5 @@
+<xunit>
```

Comment:
I don't think there are used anymore.  We can probably just delete them in mass from the repo.